symmetries.py

```diff
@@ -82,3 +83,46 @@ def randomize_symmetries_feat(features):
 def invert_symmetries_pi(symmetries, pis):
     return [apply_symmetry_pi(invert_symmetry(s), pi)
             for s, pi in zip(symmetries, pis)]
+
+
+def rotate_train(x, pi):
+    sym = tf.random_uniform(
```

Comment:
I used tf.Print to verify this took on multiple values during tests but I couldn't verify on the TPU (because op happens on the TPU)